prog/vnet/cert_nexus.rb

```diff
@@ -98,7 +98,11 @@ def before_run
 
   label def destroy
     # the reason is chosen as "cessationOfOperation"
-    acme_client.revoke(certificate: cert.cert, reason: REVOKE_REASON) if cert.cert
+    begin
+      acme_client.revoke(certificate: cert.cert, reason: REVOKE_REASON) if cert.cert
+    rescue Acme::Client::Error::AlreadyRevoked
+      Clog.emit("Certificate is already revoked")
```

Comment:
I suppose there is no useful structured data here, i.e we get enough for free from Clog.emit?

Reply:
We can print the whole exception but, yeah, there is nothing useful in there.